src/leonardo/xhr-mock.srv.ts

```diff
@@ -40,7 +40,8 @@ export class XhrMock {
       resHeaders = Utils.isFunction(activeOption.headers) ? activeOption.headers(requestEventObj) : activeOption.headers;
     }
     const delay = activeOption.delay || 0;
-    const res = response.status(resStatus).headers(resHeaders).body(resData);
+    const deepCopy = data => JSON.parse(JSON.stringify(data)) 
```

Comment:
@dwido are you sure everything is json? I think we support any response. plus, I think the unit tests ar failing.